
sensor_msgs/PointCloud2 obj_cloud
geometry_msgs/Point p_center
---
bool success
sensor_msgs/PointCloud2 ray_cast_cloud
int16[] array
float32 distance